.gitignore

```diff
@@ -1,6 +1,5 @@
 *~
 vendor
-composer.lock
```

Comment:
why did you remove `composer.lock` from the gitignore? We intentionally are ignoring this file.